CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 1.8.0
+- Change submit button and widget title default values [[#103](https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/103)]
+- Fix admin style loading [[#101](https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/101)]
+- Improve the wording of widget texts [[#100](https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/100)]
+- Implement migrations [[#98](https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/98)]
+- Implement opt-in widget front-end form logic [[#97](https://https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/97)]
+- Subscription form widget refinement [[#92](https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/92)]
+- Formatting and logic correction for Smaily subscription form [[#85](https://github.com/sendsmaily/smaily-woocommerce-plugin/issues/85)]
```

Comment:
Changes for version 1.8.0 should only reflect the changes compared to previous version.

In hindsight we should have used an epic branch for the feature development, but for now I suggest you omit #103, #101, #100, #98 from list of changes and aggregate #97, #92 and #85 into a single change log entry.